features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/js/logic/create.asset.js

```diff
@@ -265,7 +264,7 @@ $(function() {
 			success: function(response) {
                 var result = {};
                 try {
-                    result = JSON.parse(response);
+                    result = (typeof response == "string") ? JSON.parse(response) : response;
```

Comment:
Please put a comment why typeof response == "string" check was necessary. Seems there is something wrong in the service. Please consider adding JIRA if the service sends two types of responses.